tests/Functional/ContentEventRenderTest.php

```diff
@@ -0,0 +1,137 @@
+<?php
+
+declare(strict_types = 1);
+
+namespace Drupal\Tests\oe_theme\Functional;
+
+use Drupal\file\Entity\File;
+use Drupal\Tests\BrowserTestBase;
+
+/**
+ * Tests that our Event content type render.
+ *
+ * @todo: Extend this test with ecl/markup rendering tests.
+ */
+class ContentEventRenderTest extends BrowserTestBase {
+
+  /**
+   * The node storage.
+   *
+   * @var \Drupal\node\NodeStorageInterface
+   */
+  protected $nodeStorage;
+
+  /**
+   * {@inheritdoc}
+   */
+  public static $modules = [
+    'config',
+    'system',
+    'oe_theme_helper',
+    'path',
+    'oe_theme_content_event',
+    'content_translation',
+    'oe_multilingual',
+  ];
+
+  /**
+   * {@inheritdoc}
+   */
+  protected function setUp() {
+    parent::setUp();
+
+    // Enable and set OpenEuropa Theme as default.
+    $this->container->get('theme_installer')->install(['oe_theme']);
+    $this->container->get('theme_handler')->setDefault('oe_theme');
+
+    // Rebuild the ui_pattern definitions to collect the ones provided by
+    // oe_theme itself.
+    $this->container->get('plugin.manager.ui_patterns')->clearCachedDefinitions();
+
+    $this->nodeStorage = $this->container->get('entity_type.manager')->getStorage('node');
+  }
+
+  /**
+   * Tests that the Event featured media renders the translated media.
+   */
+  public function testEventFeaturedMediaTranslation(): void {
+    // Make event node and image media translatable.
+    $this->container->get('content_translation.manager')->setEnabled('node', 'oe_event', TRUE);
+    $this->container->get('content_translation.manager')->setEnabled('media', 'image', TRUE);
+    $this->container->get('router.builder')->rebuild();
+
+    // Create image media that we will use for the English translation.
+    $this->container->get('file_system')->copy(drupal_get_path('theme', 'oe_theme') . '/tests/fixtures/example_1.jpeg', 'public://example_1.jpeg');
+    $en_image = File::create([
+      'uri' => 'public://example_1.jpeg',
+    ]);
+    $en_image->save();
+
+    // Create image media that we will use for the Bulgarian translation.
+    $this->container->get('file_system')->copy(drupal_get_path('theme', 'oe_theme') . '/tests/fixtures/placeholder.png', 'public://placeholder.png');
+    $bg_image = File::create([
+      'uri' => 'public://placeholder.png',
+    ]);
+    $bg_image->save();
```

Comment:
Let's use the same as we use below, for consistency and it's also more appropriate (with the _en,  _bg etc.):

```php
    $en_file = file_save_data(file_get_contents(drupal_get_path('theme', 'oe_theme') . '/tests/fixtures/example_1.jpeg'), 'public://example_1_en.jpeg');
    $en_file->setPermanent();
    $en_file->save();

    // Create Bulgarian file.
    $bg_file = file_save_data(file_get_contents(drupal_get_path('theme', 'oe_theme') . '/tests/fixtures/example_1.jpeg'), 'public://example_1_bg.jpeg');
    $bg_file->setPermanent();
    $bg_file->save();
```